controllers/autoscaling.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/actions-runner-controller/actions-runner-controller/api/v1alpha1"
+	"github.com/google/go-github/v37/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v39/github"
```

This seems outdated since #925. I'll update your commit with the change.